tests/frontend/unit/DpAutofillSubmitterData.spec.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * (c) 2010-present DEMOS E-Partizipation GmbH.
+ *
+ * This file is part of the package demosplan,
+ * for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+import shallowMountWithGlobalMocks from '@DpJs/VueConfigLocal'
+
+// Component to test
+import SubmitterComponent from '@DpJs/components/statement/statement/DpAutofillSubmitterData'
+
+// Test data which is defined in Twig
+import Submitters from './DpAutofillSubmitterData.json'
+
+describe('Submitter', () => {
+  it('should be an object', () => {
+    expect(typeof SubmitterComponent).toBe('object')
+  })
+
+  it('should be named DpAutofillSubmitterData', () => {
+    expect(SubmitterComponent.name).toBe('DpAutofillSubmitterData')
+  })
+
+  //  This represents permissions for Bob-HH + Bob-SH
```

Comment:
Should we remove this comment?

Reply:
yes. absolutely ebea42bf8028deab62c33b12c886a13acfdec18f